### DISCUSSION TITLE
Candidatura di: Stefano Biancardo

### DISCUSSION THREAD

**Discussion by Cadienvan** [Il-Libro-Open-Source/book#40] [Candidature Ambassadors In Corso]
Questo sondaggio viene aperto per tutto il mese di Settembre per indicare la preferenza o l'opposizione sulla candidatura a ruolo di Ambassador per:  
### [Stefano Biancardo](https://github.com/Sab-tech-lab)

In caso di disaccordo, si protegge l'anonimato ma sarebbe gradita una spiegazione tramite commento a questo sondaggio con delle motivazioni.  

Attualmente, vista la fase embrionale del progetto, non ci saranno limiti minimi di votazione e la semplice superiorità dei voti a favore sarà considerata sufficiente a procedere con la nomina.

**Comment by eppak** [comment]
Ciao @Sab-tech-lab, 
Ti confermiamo che la votazione si chiude oggi e che è positiva. Benvenuto tra gli Ambassador, ti chiediamo di aggiungere Michael su Telegram «at»Cadienvan che provvederà ad aggiungerti al gruppo organizzativo. 
Ti ricordiamo di prendere visione, oltre delle linee guida, anche del nostro codice di condotta che trovi https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md. 
Il repository sulla governance invece lo trovi a questo indirizzo dove sono presenti anche i gruppi di lavoro. 
Ti aspettiamo!

**Comment by eppak** [comment]
Ciao Sab-tech-lab, ti ricordiamo che per confermare la tua partecipazione al team Ambassador, ti basta aggiungere Michael su Telegram «at» Cadienvan che provvederà ad aggiungerti al gruppo organizzativo.
Ti ricordiamo di prendere visione, oltre delle linee guida, anche del nostro codice di condotta che trovi https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md.
Ti aspettiamo!

P.S. hai ancora fino a Domenica 15 Ottobre per confermare la tua partecipazione, dopodiché la mancanza di comunicazione verrà considerata una mancata accettazione di questo invito.

**Comment by Sab-tech-lab** [comment]
Ciao Alessandro,
Purtroppo per impegni di lavoro non ho più molto tempo e non riuscirei a dedicare il giusto tempo e la giusta attenzione al progetto.

Buon lavoro

**Reply by eppak** [reply]
Casomai volessi contribuire siamo qui.
Grazie mille per l'aggiornamento!

**Reply by Sab-tech-lab** [reply]
Certo se potrò essere di aiuto in qualche modo lo farò più che volentieri!
